VideoLocker/res/values/styles.xml

```diff
@@ -83,14 +83,14 @@
     <style name="last_access_bar_style">
         <item name="android:layout_width">match_parent</item>
         <item name="android:layout_height">75dp</item>
-        <item name="android:background">@color/cyan_1</item>
+        <item name="android:background">@color/edx_brand_primary_x_light</item>
         <item name="android:visibility">gone</item>
     </style>
 
     <style name="download_in_progress_bar_style">
         <item name="android:layout_width">match_parent</item>
         <item name="android:layout_height">45dp</item>
-        <item name="android:background">@color/white</item>
+        <item name="android:background">@color/edx_brand_primary_x_light</item>
```

Comment:
This style should inherit from `last_access_bar_style`, or vice versa, or both should inherit from a base style. The only difference between them now is the height.